ParseTwitterUtils/PF_Twitter.m

```diff
@@ -92,6 +92,30 @@ - (BFTask *)authorizeInBackground {
         return source.task;
     }];
 }
+- (BFTask *)deauthorizeInBackground {
+    if (self.consumerKey.length == 0 || self.consumerSecret.length == 0) {
+        //TODO: (nlutsenko) This doesn't look right, maybe we should add additional error code?
+        return [BFTask taskWithError:[NSError errorWithDomain:PFParseErrorDomain code:1 userInfo:nil]];
+    }
+
+    return [[self _performDeauthAsync] pftw_continueAsyncWithBlock:^id(BFTask *task) {
+        BFTaskCompletionSource *source = [BFTaskCompletionSource taskCompletionSource];
+        dispatch_async(dispatch_get_main_queue(), ^{
+            if (task.cancelled) {
+                [source cancel];
+            } else if (!task.error && !task.result) {
+                source.result = nil;
+            } else if (task.error) {
+                [source trySetError:task.error];
+            } else if (task.result) {
+                [self setLoginResultValues:@{}];
```

Comment:
I went with passing in an empty dictionary literal since it's nice to not have an optional when dealing with Swift, and, well, it's a cheap operation (iirc, it's like one extra `mov` instruction compared to passing in `nil`).

that said, i don't feel that too strongly about this, and can revert if you would prefer ¯\_(ツ)_/¯